packages/databyss-ui/modules/Sidebar/Sidebar.js

```diff
@@ -133,6 +72,9 @@ const Sidebar = () => {
         </PagesLoader>
       )
     }
+    if (menuItem === 'search') {
+      return null
+    }
     return SidebarList({
```

Comment:
We should migrate `SidebarList` to `routes/Pages` to make the handling consistent.